src/components/Activists/Member.jsx

```diff
@@ -49,85 +61,190 @@ const Member = ({ member, handleClose }) => {
+                borderRadius: 2,
+              }}
+            >
+              <Typography component="h4" sx={{ color: 'colors.gray.700', fontSize: '14px', mb: 1, fontWeight: '500' }}>
+                Informations personnelles
+              </Typography>
+              <LineText label="Adresse complète" value={`${member.raw.address}, ${member.city} ${member.postalCode}`} />
+              <Grid container spacing={1}>
+                <Grid item xs={12} sm={6}>
+                  <LineText label="Nationalité" value={member.raw.nationality} />
+                </Grid>
+                <Grid item xs={12} sm={6}>
+                  <LineText label="Sexe" value={member.gender === 'male' ? 'Homme' : 'Femme'} />
+                </Grid>
+              </Grid>
+            </Box>
+
+            <Box
+              sx={{
+                bgcolor: 'colors.gray.50',
+                px: 2,
+                py: 2.5,
+                borderRadius: 2,
+              }}
+            >
+              <Typography component="h4" sx={{ color: 'colors.gray.700', fontSize: '14px', mb: 1, fontWeight: '500' }}>
+                Informations globales
+              </Typography>
+              {member.committee && <LineText label="Comité" value={member.committee} />}
+              <Grid container spacing={1}>
+                <Grid item xs={12} sm={6}>
+                  <LineText
+                    label="Renaissance"
+                    value={
+                      <MemberBadge membership={member.raw.renaissance_membership} labelStyle={{ fontSize: '14px' }} />
+                    }
+                  />
+                </Grid>
+                <Grid item xs={12} sm={6}>
+                  <LineText label="Date d'inscription" value={format(member.joinedDate, 'dd/MM/yyyy')} />
+                </Grid>
+              </Grid>
+              <Box
+                sx={{
+                  borderTop: '1px solid',
+                  borderTopColor: 'colors.gray.200',
+                  pt: 0.5,
+                  mt: 1,
+                }}
+              >
+                <LineText
+                  label="Intérêts"
+                  value={
+                    <Box display="flex" alignItems="center" flexWrap="wrap" className="space-x-2 space-y-1">
+                      {member.interests.length > 0 &&
+                        member.interests.map((interest, index) => (
+                          <UIChip
+                            key={index}
+                            label={interest}
+                            color="colors.blue.500"
+                            bgcolor="colors.blue.50"
+                            labelStyle={{ fontSize: '14px' }}
+                          />
+                        ))}
+                      {member.interests.length === 0 && (
+                        <Typography component="p" sx={{ color: 'colors.gray.900' }}>
+                          Aucun
+                        </Typography>
+                      )}
```

Comment:
au lieu de refaire la condition sur `.length` tu peux faire un `||` 
`[].length > 0 && <Foo /> || <Bar />`

```suggestion
                      {member.interests.length > 0 &&
                        member.interests.map((interest, index) => (
                          <UIChip
                            key={index}
                            label={interest}
                            color="colors.blue.500"
                            bgcolor="colors.blue.50"
                            labelStyle={{ fontSize: '14px' }}
                          />
                        )) ||  (
                        <Typography component="p" sx={{ color: 'colors.gray.900' }}>
                          Aucun
                        </Typography>
                      )}
```